.rubocop.yml

```diff
@@ -1,88 +1,123 @@
-# Rubocop configuration
-
+---
+require: rubocop-rspec
 AllCops:
+  DisplayCopNames: true
+  TargetRubyVersion: '2.1'
+  Include:
+  - "./**/*.rb"
   Exclude:
-    - 'pkg/**/*'
-    - 'vendor/**/*'
-
-# Baseline code complexity metrics - we should try and reduce these over time
-Metrics/AbcSize:
-  Max: 47
-
-Metrics/CyclomaticComplexity:
-  Max: 16
-
```

Comment:
Can you provide the rationale for removing our existing settings? We established these several years ago and we'd like to understand why you're proposing changing these now. Note that we're not necessarily against some of these changes. 

Please provide a short justification for each cop you're removing. We can discuss in more detail if necessary.

Reply:
As part of the `pdk conversion` it uses the same cops that are used across Puppet modules to ensure consistency. We can add back in these cops through the .sync.yml

Reply:
@Thomas-Franklin Thanks.  Is this something that can be added as part of this PR so we can review?